lib/hushed/blackboard.rb

```diff
@@ -40,5 +42,40 @@ def build_document(type, contents)
       end
       namespace.const_get(type).new(:io => contents) if namespace
     end
+
+    def retrieve_latest(document_name)
+      blackboard = client.from_quiet_bucket
+      documents = inventory_summary_documents(blackboard, document_name)
```

Comment:
This method name (inventory_summary_documents) is confusing since this is not specific to inventory summary.  Shouldn't this be `documents_starting_with` or `documents_with_prefix` or something like that.